test/langtools/tools/javac/util/StringUtilsTest.java

```diff
@@ -66,4 +119,10 @@ void assertEquals(int expected, int actual) {
             throw new IllegalStateException("expected=" + expected + "; actual=" + actual);
         }
     }
+
+    void assertTrue(boolean cond) {
+        if (!cond) {
+            throw new IllegalStateException();
```

Comment:
Mildly surprising to see `IllegalStateException` in `assert...` methods.

Reply:
I simply followed the style of the file.